src/components/session/NewSessionSetup.vue

```diff
@@ -47,7 +47,7 @@ export default {
 
 <template>
   <div class="w-100 h-100 card">
-    <div class="m-auto w-50 h-auto card">
+    <div class="m-auto h-auto card" style="min-width: 75%">
```

Comment:
Wouldn't replacing `w-50` with `w-75` have the same effect?

Reply:
note that we specify `min-height`, enabling the container to be bigger

bootstrap only has classes for normal and max height iirc, feel free to correct me

Reply:
I suppose I'm not quite understanding then, where do we specify `min-height`?

Reply:
the line you linked. Its a normal css style property